### ISSUE TITLE
Generated icon is smaller than default Adwaita

### ISSUE THREAD

**Issue by skewty** [alex285/Adwaita-Supplementary-Folders#1]
When running Adwaita on Ubuntu 19.04 using gnome vanilla package.  The icons generated are smaller than the other icons.

![Screenshot from 2019-06-09 09-49-19](https://user-images.githubusercontent.com/9087223/59161833-54939480-8a9c-11e9-9626-f93511212fc0.png)

I took some screenshots and was comparing the size difference in GIMP.  Here it is:
![Screenshot from 2019-06-09 09-53-04](https://user-images.githubusercontent.com/9087223/59161852-7c82f800-8a9c-11e9-981a-c191a8b3a1be.png)

Are the icons not supposed to be the same size?  Is Canonical not using the proper upstream Adwaita icons or ???

**Comment by skewty** [comment]
So I noticed that the icon is smaller only at the smallest zoom level in Nautilus.

**Comment by alex285** [comment]
right! i got that on README todo, but i completely forgot to fix it, because i dont resize them; i will created the smaller sizes in a few days! sorry!
